src/esw/drive/odrive_control.py

```diff
@@ -623,27 +635,51 @@ def run(self):
 
         rospy.spin()
 
-    def _drive_vel_cmd_callback(self, ros_msg: JointState, axis: Axis) -> None:
+    def _process_twist_message(self, ros_msg: Twist):
+        """Converts the twist message into rad/s to turn per wheel.
+        Then tells the wheels to move at that speed.
+        :param ros_msg: A Twist message of the rover to turn."""
+        forward = ros_msg.linear.x
+        turn = ros_msg.angular.z
+
+        turn_difference = turn * self._wheel_base / 2
+
+        left_m_s = forward + turn_difference
+        right_m_s = forward - turn_difference
+
+        if (abs(left_m_s) > self._max_speed_m_s) or (abs(right_m_s) > self._max_speed_m_s):
```

Comment:
So right now ESW code limits speed in two ways:
1) You can change the max meters per second of the rover in m/s in the config (currently 0.8 m/s)
2) The motors themselves are capped at 50 revolutions per second (in ESW code and can NOT be changed in config).

IDK if we want to keep method 1. I can only see two reasons for why it would be useful:
1) Give a reasonable expectation to the user of how fast the rover is going.
2) If we want to to make the rover+motors capped out a slower speed than what we allow the motors to move at.